x11-misc/quitcount/quitcount-3.1.4.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg
+
+# The string in SRC_URI doesn't follow standard naming convention
+MV="3.1"
+DESCRIPTION="A simple applet that shows what you saved since you quit smoking"
+HOMEPAGE="http://quitcount.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${MV}/${P}.tar.gz"
```

Comment:
```suggestion
DESCRIPTION="A simple applet that shows what you saved since you quit smoking"
HOMEPAGE="http://quitcount.sourceforge.net/"
SRC_URI="mirror://sourceforge/${PN}/$(ver_cut 1-2)/${P}.tar.gz"
```

Should work.